js/src/dropdown.js

```diff
@@ -192,10 +187,10 @@ const Dropdown = (($) => {
           continue
         }
 
-        // remove backdrop only if the dropdown menu will be hidden
-        const backdrop = $(parent).find(Selector.BACKDROP)[0]
-        if (backdrop) {
-          backdrop.parentNode.removeChild(backdrop)
+        // if this is a touch-enabled device we remove the extra
+        // empty mouseover listeners we added for iOS support
+        if ('ontouchstart' in document.documentElement) {
+          $('body').children().off('mouseover', $.noop)
```

Comment:
although this is now merged and closed ... i actually got a follow-up here: it seems that even without the selector, it works as expected. when i ran your test in chrome just now, once i did the `$('body').children().off('mouseover', lala)` it actually removed the lala function correctly (so only toto was triggered).

further, am i right in thinking that `$('body').children()` and `$('body').children().find("*")` (which would be what is selected using your proposed change) are not the same thing, as i'm going one level deeper?

(asking because i'm about to add this exact same tweak to tooltip.js, and it made me look deeper into jquery, which is not usually my thing)